### ISSUE TITLE
[Bug]: Crash with transmutation table

### ISSUE THREAD

**Issue by Hammer512** [FTBTeam/FTB-Modpack-Issues#1710]
### Modpack

Ftb Stoneblock 3

### Modpack version

1.1.0

### log files
[latest.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/9966584/latest.log)

_No response_

### Describe the bug

I have a emc farm with the power flowers(specifically pink) and whenever i use the watch of flowing time to speed up said power flowers, everything is fine untill i open my transmutation table and my game just crashes.

### Steps to reproduce

place down about 50 pink power flowers in a area
put a watch of flowing time on a pedastal fully charged and set to fast-forward place 2 of them so they cover every single power flower
open a transmutation table

### Expected behaviour

i expected for my emc to get boosted a bunch and for nothing else to happen

**Comment by Hammer512** [comment]
where do i find my crash log?

**Comment by Raidobw2** [comment]
Click the "..." button in the modpack overview page on the FTB App, then you can open the logs and crash reports folders from there.

Or you can open "instance folder" from there, and you'll see the folders `logs` and `crash-reports`, which both contain the errors generally. You can add both a crash report and the `latest.log` if you want

**Comment by Hammer512** [comment]
i do not see the crash-reports thing even when i search it up

**Comment by Raidobw2** [comment]
The latest.log in the logs folder would suffice in that case, although right now Mojang is having problems with authentification so you might have difficulties launching the modpack on the FTB App

**Comment by Hammer512** [comment]
[latest.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/9966584/latest.log)

**Comment by Hammer512** [comment]
> Crash log please

[latest.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/9966584/latest.log)

**Comment by Raidobw2** [comment]
That log doesn't have the error unfortunately... Is this the log from your latest play session where this happened? Otherwise all files next to `latest.log` are also logs (but archived), if you recall the approximate time frame for that day, you could zip all those logs from the day before this happened until after this happened and put them here, I'll sort them to figure out if there's an error hiding somewhere. Thanks🙂

**Comment by Thunder-Nova** [comment]
Same happened to my friend, I have what seems to be a better PC then him perhaps in terms of graphic card. We both have the same 10 000MB Ram allocated.

So I told him to pass me a book to share knowledge and I had to type in search pipe. I got some lag but no crash, yet for him, he does crash. (I had no knowledge saved in my transmutation tablet)

So this issue might be related to the ProjectE mod itself, plus perhaps how much ram allocated.

I have this Java Custom Argument, so might try that, though idk if it will help

`-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms256m`
